_docs/reference/release-roadmap.md

```diff
@@ -8,5 +8,28 @@ layout: docs
 type: markdown
 ---
 
-TBD
+## Istio v0.2
 
+Istio v0.2 will be our second release, and will work across multiple Kubernetes clusters and raw VMs, with a larger number of services. Istio v0.2 will support the following features:
+- TCP telemetry collection and policy enforcement.
+- Hybrid deployments (multiple clusters, VMs).
```

Comment:
Are you guys committed to adding CF support? I was trying to leave it generic until we had finalized order in which we add new environments...